manim/_config/main_utils.py

```diff
@@ -453,4 +458,10 @@ def _parse_args_no_subcmd(args: list) -> argparse.Namespace:
         metavar="True/False",
     )
 
+    parser.add_argument(
+        "--plugins",
+        type=_split,
+        help="Enable plugins",
+    )
+
```

Comment:
For some reason, this doesn't work ie) doesn't give the list exactly when it is imported...
@leotrs can you help me here?

Reply:
What does it give you and what do you expect instead?

Reply:
Maybe because argparse isn't initialised? If this is the problem then I think `--plugins` command line parameter would be very hard to implement.

Reply:
After first sight, I think you may want to change this to:
```python
    parser.add_argument(
        "--plugins",
        nargs='+',
        type=str,
        help="Enable plugins",
    )
```
    
Then you should be able to run this using:
manim basic.py --plugins manim_plugintemplate manim_rubikscube manim_otherplugin

Reply:
I haven't tested this, but I'll fetch your repo and give it a shot momentarily. Also, as an aside, I've managed to figure out how to make the custom decorators necessary for #452 in transferring to click

Reply:
After looking at this more closely, this is not an issue with the parser phrase, but instead an issue with where you're placing the print statement. Keep in mind that the file you're running is imported and this is run first thing before any of the argparse logic hits. The arg parse logic is only evaluated after the imports. In particular:
https://github.com/naveen521kk/manim/blob/plugins/manim/__main__.py#L78

Placing a print statement after this line will show that the plugins are placed under `config['plugins']` as expected; however, this is only after the config.digest_args.

Reply:
Now this makes me think that this will not work directly. So I think it will be better to remove `--plugins` CLI instead use only from the Config file for now.

Reply:
What if you moved the contents of this file into a function and called the function after config.digest_args(args)? Would this not work?

Reply:
No. Here `__all__` is manipulated and wildcart imports isn't possible from within a function.

Reply:
I am removing the option to set plugins using the command line.